src/apps/content-editor/src/app/components/Editor/Field/Field.tsx

```diff
@@ -703,11 +703,15 @@ export const Field = ({
 
     case "one_to_one":
       const onOneToOneOpen = useCallback(() => {
-        return dispatch(
-          fetchItems(relatedModelZUID, {
-            lang: getSelectedLang(allLanguages, langID),
-          })
-        );
+        if (!!relatedModelZUID) {
```

Comment:
What are we confirming with this check? That it is not blank? It is not `0`? It not `null`? Would this be clear as `Boolean()`? Should this be using our https://github.com/zesty-io/zuid-js lib to validate the ZUID?

Reply:
Yeah, it's just checking if it has a value. I wasn't aware that we have a zuid validator I could make use, I guess it makes sense to be using that validator instead.

Reply:
@shrunyan Updated to use `zuid.isValid`